src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -38,28 +43,36 @@ const defaultProps = {
 const ReportActionItemCreated = (props) => {
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
+
     return (
         <OfflineWithFeedback
             pendingAction={lodashGet(props.report, 'pendingFields.addWorkspaceRoom') || lodashGet(props.report, 'pendingFields.createChat')}
             errors={lodashGet(props.report, 'errorFields.addWorkspaceRoom') || lodashGet(props.report, 'errorFields.createChat')}
             errorRowStyles={styles.addWorkspaceRoomErrorRow}
             onClose={() => Report.navigateToConciergeChatAndDeleteReport(props.report.reportID)}
         >
+            <View pointerEvents="none" style={[styles.emptyStateBackgroundContainer, StyleUtils.getBackgroundImageHeightStyle(props.isSmallScreenWidth)]}>
+                <Image
```

Comment:
Yeah I went that direction initially, but because the content itself doesn't extend up as high as the background needed to, I was finding it close to impossible to get it to look right and not get cut off at the top. I'll try a little more today, but I'm not hopeful (trust me, I would definitely prefer to use `ImageBackground` if it would work)